src/Providers/ExtensionsServiceProvider.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Carnival\Providers;
+
+use Carnival\Extension\ExtensionManager;
+use Illuminate\Support\ServiceProvider;
+
+class ExtensionsServiceProvider extends ServiceProvider {
+    public function register() {
+        $this->app->bind('extensions', fn() => new ExtensionManager);
```

Comment:
This is clean I am going to update the hook system to bind the same way.

Reply:
Gotta love arrow functions 😀